flink-libraries/flink-cep/src/test/java/org/apache/flink/cep/operator/CEPMigrationTest.java

```diff
@@ -53,9 +54,14 @@
  * Tests for checking whether CEP operator can restore from snapshots that were done using previous
  * Flink versions.
  *
+ * <p>NOTE: Due to major upgrades of serializers, Flink 2.0 is not compatible with saved state from
+ * Flink 1.x, and this test is temporarily disabled. This test should be reinstated for future
+ * versions of Flink, such as Flink 2.1, that promise compatibility with older releases.
+ *
  * <p>For regenerating the binary snapshot file of previous versions you have to run the {@code
  * write*()} method on the corresponding Flink release-* branch.
  */
+@Ignore("Flink 2.0 is not compatible with saved state from Flink 1.x")
```

Comment:
We need a blocker ticket for 2.1 for this

Reply:
Same question. Is this good?
https://issues.apache.org/jira/browse/FLINK-37404

Reply:
looks good!